src/utils/index.js

```diff
@@ -0,0 +1,49 @@
+const { filterType, filterComponents } = require('./filters');
```

Comment:
Instead of doing multiple checks, can we generalize this to parse and return the values?

For example; Given `S Main Street`; this is passed into a Func then returns it as something like:

```
{
   preDirection: 'S',
   postDirection: null,
   address: 'Main Street',
}
```

Then use the return values in the `getPostDirection` `getPreDirection` functions above.